diff -urN gnutu-2.1.org/gnutu.desktop.in gnutu-2.1/gnutu.desktop.in
--- gnutu-2.1.org/gnutu.desktop.in	2005-10-29 20:50:16.747607000 +0200
+++ gnutu-2.1/gnutu.desktop.in	2005-10-29 20:51:04.057607000 +0200
@@ -7,8 +7,7 @@
 Name[bg]=Програма за ученици
 TryExec=gnutu
 Exec=gnutu
-Icon=@prefix@/share/pixmaps/gnutu.png
+Icon=gnutu.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Utility;
-GenericName[es_ES]=
+Categories=GTK;GNOME;Application;Utility;
