diff -urN gnutu-1.0.orig/gnutu.desktop gnutu-1.0/gnutu.desktop
--- gnutu-1.0.orig/gnutu.desktop	2004-11-12 20:45:01.955779648 +0100
+++ gnutu-1.0/gnutu.desktop	2004-11-12 20:46:24.473235088 +0100
@@ -5,7 +5,8 @@
 Comment[pl]=GNU Terminarz Ucznia - wersja 1.0
 TryExec=gnutu
 Exec=gnutu
-Icon=/usr/local/share/pixmaps/gnutu/gtu_ikonka.png
+Icon=gtu_ikonka
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Utility;
+Categories=GTK;GNOME;Utility;
+Encoding=UTF-8
